demo_serial/src/main/java/com/novoda/downloadmanager/demo/serial/MainActivity.java

```diff
@@ -41,21 +41,22 @@ protected void onCreate(Bundle savedInstanceState) {
 
     private void setupDownloadingExample() {
         Uri uri = Uri.parse(BIG_FILE);
-        final DownloadBatch batch = new DownloadBatch("Large Beard Shipment", "Goatees galore", BEARD_IMAGE);
+        final RequestBatch batch = RequestBatch.newInstance("Large Beard Shipment", "Goatees galore", BEARD_IMAGE);
```

Comment:
so you should probably create a `BatchExampleActivity` and reset this demo back to the standard case of a download.  (or make this a menu to other activities)